src/redux/selectors/select-prop-from-data.js

```diff
@@ -1,3 +1,5 @@
+import { get } from 'lodash';
+
 export default function selectPropFromData(store, type) {
-  return store.eholdings.data[type];
+  return get(store, `eholdings.data.${type}`, {});
```

Comment:
Just suggestion, it would be better to omit using of lodash, it could be looks like
```suggestion
  return store?.eholdings?.data?[type] || {};
```
but, all is fine, it has the right to live